sdm.tf

```diff
@@ -0,0 +1,13 @@
+resource "sdm_node" "gateway" {
```

Comment:
It would be beneficial to include the `name` attribute here to help differentiate these. Something like I did in the concourse version.
https://github.com/highwingio/concourse/blob/main/modules/sdm_gateway/sdm_gateway/main.tf

Reply:
Good call, done in https://github.com/highwingio/terraform-aws-strongdm/pull/9/commits/d440ca0dc1e3d24d0be789d38c7b3ebecff5c8be